ecr-login/vendor/golang.org/x/sys/unix/mkasm_darwin.go

```diff
@@ -1,61 +0,0 @@
-// Copyright 2018 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-// +build ignore
-
-// mkasm_darwin.go generates assembly trampolines to call libSystem routines from Go.
-//This program must be run after mksyscall.go.
```

Comment:
These removed files have "+build ignore" and apparently used as code generators. Removing them shouldn't affect this package.

Reply:
https://github.com/golang/sys/blob/master/unix/README.md#mksyscallgo